diff --git SDL2-2.26.5/include/SDL_stdinc.h SDL2-2.26.5/include/SDL_stdinc.h
index bbce3d0..57dfd46 100644
--- SDL2-2.26.5/include/SDL_stdinc.h
+++ SDL2-2.26.5/include/SDL_stdinc.h
@@ -712,6 +712,8 @@ extern DECLSPEC char *SDLCALL SDL_iconv_string(const char *tocode,
    here, since there are possibly tests that are ineffective otherwise. */
 #if defined(__clang_analyzer__) && !defined(SDL_DISABLE_ANALYZE_MACROS)
 
+#include <string.h> // silencing clang-diagnostic-error: "use of undeclared identifier 'memcpy'"
+
 /* The analyzer knows about strlcpy even when the system doesn't provide it */
 #ifndef HAVE_STRLCPY
 size_t strlcpy(char* dst, const char* src, size_t size);
