diff --git a/CMakeLists.txt b/CMakeLists.txt
index 87d309c2..d5c1e4fa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -271,7 +271,7 @@ endif()
 
 target_compile_definitions(datachannel PRIVATE RTC_EXPORTS)
 
-add_library(datachannel-static STATIC EXCLUDE_FROM_ALL
+add_library(datachannel-static STATIC
 	${LIBDATACHANNEL_SOURCES}
 	${LIBDATACHANNEL_HEADERS}
 	${LIBDATACHANNEL_IMPL_SOURCES}
@@ -445,6 +445,13 @@ install(TARGETS datachannel EXPORT LibDataChannelTargets
 	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
 
+install(TARGETS datachannel-static EXPORT LibDataChannelStaticTargets
+#install(TARGETS usrsctp plog juice-static datachannel-static EXPORT LibDataChannelStaticTargets
+	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+	LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
+
 install(FILES ${LIBDATACHANNEL_HEADERS}
 	DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/rtc
 )
@@ -456,12 +463,22 @@ install(
 	NAMESPACE LibDataChannel::
 	DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/LibDataChannel
 )
+#install(
+#	EXPORT LibDataChannelStaticTargets
+#	FILE LibDataChannelStaticTargets.cmake
+#	NAMESPACE LibDataChannel::
+#	DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/LibDataChannelStatic
+#)
 
 # Export config
 install(
 	FILES ${CMAKE_CURRENT_SOURCE_DIR}/cmake/LibDataChannelConfig.cmake
 	DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/LibDataChannel
 )
+install(
+	FILES ${CMAKE_CURRENT_SOURCE_DIR}/cmake/LibDataChannelConfig.cmake
+	DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/LibDataChannelStatic
+)
 
 # Export config version
 include(CMakePackageConfigHelpers)
@@ -471,6 +488,8 @@ write_basic_package_version_file(
 	COMPATIBILITY SameMajorVersion)
 install(FILES ${CMAKE_BINARY_DIR}/LibDataChannelConfigVersion.cmake
 	DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/LibDataChannel)
+install(FILES ${CMAKE_BINARY_DIR}/LibDataChannelConfigVersion.cmake
+	DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/LibDataChannelStatic)
 
 # Tests
 if(NOT NO_TESTS)
